core/whats-new/whats-new-in-21/cs/Program.cs

```diff
@@ -13,6 +13,8 @@ static void Main()
         var part = new Span<int>(numbers, start: 10, length: 10);
         foreach (var value in part)
             Console.Write($"{value}  ");
+
+        Console.ReadLine();
```

Comment:
We generally don't include calls to Console.ReadLine; it's useful when running in debug mode in Visual Studio, but not when run outside of it. (And without a message, the need to press Enter to end the example is not obvious.) A call to Console.WriteLine makes sense, though, since the previous statement calls Console.Write.